layout/formlayout.go

```diff
@@ -126,23 +120,19 @@ func (f *formLayout) Layout(objects []fyne.CanvasObject, size fyne.Size) {
 // For a FormLayout this is the width of the widest label and content items and the height is
 // the sum of all column children combined with padding between each.
 func (f *formLayout) MinSize(objects []fyne.CanvasObject) fyne.Size {
-	table := f.tableCellsSize(objects, 0)
+	labelWidth, contentWidth, heights := f.tableCellsSize(objects, 0)
+	if len(heights) == 0 {
+		return fyne.Size{}
+	}
 
 	padding := theme.Padding()
-	minSize := fyne.NewSize(0, 0)
-
-	if len(table) == 0 {
-		return minSize
+	minSize := fyne.Size{
+		Width:  labelWidth + contentWidth + padding,
+		Height: padding * float32(len(heights)-1),
```

Comment:
nit: could just inline `theme.Padding()` call here since it's only used once

Reply:
I tried that as well but look on the line above and you will see that that isn't used once.